cloudflare/eslint.config.mjs

```diff
@@ -0,0 +1,18 @@
+import eslint from "@eslint/js";
+import tseslint from "typescript-eslint";
+
+export default tseslint.config(
```

Comment:
Is it not possible to use the top-level eslint.config.mjs instead of duplicating it here?

Reply:
Feel free to leave it if it's auto-generated.